cookbook/workflows/v2/basic_sequence_workflow.py

```diff
@@ -0,0 +1,95 @@
+from agno.agent import Agent
+from agno.models.openai import OpenAIChat
+from agno.storage.sqlite import SqliteStorage
+from agno.team import Team
+from agno.tools.googlesearch import GoogleSearchTools
+from agno.workflow.v2.sequence import Sequence
+from agno.workflow.v2.task import Task
+from agno.workflow.v2.trigger import ManualTrigger
+from agno.workflow.v2.workflow import Workflow
+
+# Define agents
+blog_analyzer = Agent(
+    name="Blog Analyzer",
+    model=OpenAIChat(id="gpt-4o"),
+    tools=[GoogleSearchTools()],
+    instructions="Extract key insights and content from blog posts",
+)
+
+content_planner = Agent(
+    name="Content Planner",
+    model=OpenAIChat(id="gpt-4o"),
+    instructions="Create engaging social media content plans based on analysis",
+)
+
+# Define research team for complex analysis
+research_team = Team(
+    name="Research Team",
+    mode="coordinate",
+    members=[blog_analyzer, content_planner],
+    instructions="Analyze content and create comprehensive social media strategy",
+)
+
+# Define tasks with consistent query-based input
+analyze_blog_task = Task(
+    name="analyze_blog",
+    agent=blog_analyzer,
+    description="Analyze the provided topic and extract key insights",
+    # No expected_input needed - automatically handles any input
+)
+
+plan_content_task = Task(
+    name="plan_content",
+    agent=content_planner,
+    description="Create social media content plan based on the research topic and previous analysis",
+    # Automatically receives outputs from previous tasks
+)
+
+research_task = Task(
+    name="research_content",
+    team=research_team,
+    description="Deep research and analysis of content",
+    # Handles any input format automatically
+)
+
+# Define sequences
+content_creation_sequence = Sequence(
```

Comment:
For later version we should allow you to skip this step if there is only one task, i.e. you can put a task into `sequences` and we'll just wrap it for you into a default sequence with the same name/description etc. 
Just could be a nicer DX

Reply:
valid observation, putting in the v1 list